manifest.toml

```diff
@@ -48,8 +48,8 @@ ram.runtime = "70M"
 
 [resources]
     [resources.sources.main]
-    url = "https://github.com/node-red/node-red/releases/download/3.1.4/node-red-3.1.4.zip"
-    sha256 = "441c17d5a5bf6e19e2ea9eec3bc1e9af572230c47eee518c2dbb2e7374137f06"
+    url = "https://github.com/node-red/node-red/archive/refs/tags/3.1.5.tar.gz"
```

Comment:
OK, I got what is wrong:
 - the auto-updater used to select the wrong asset until `autoupdate.asset = "node-red-.*zip"` was added;
 - https://github.com/node-red/node-red/releases/tag/3.1.5 does not contain such asset

For now, probably worth to close, the next minor release will probably have the correct asset proposed for download.

Reply:
No worry to have :)